src/executors/ClingExecutor.ts

```diff
@@ -0,0 +1,61 @@
+import NonInteractiveCodeExecutor from './NonInteractiveCodeExecutor';
+import * as child_process from "child_process";
+import type {ChildProcessWithoutNullStreams} from "child_process";
+import type {Outputter} from "src/Outputter";
+import type {ExecutorSettings} from "src/settings/Settings";
+
+export default class ClingExecutor extends NonInteractiveCodeExecutor {
```

Comment:
Could this be made abstract, since we shouldn't be instantiating it?

Reply:
Done! 👍